### PR TITLE
New full arrow buttons

### LINKED ISSUE #5790: Create larger hover state for prev/next buttons
Create a hover state for previous/next buttons (not that obvious the whole section is clickable) Sabrina [revised the design sketch files](https://share.goabstract.com/c5b5156f-9bb9-4147-a45d-0bf020c0c76b?collectionLayerId=442fbbfa-febf-40ce-b3ff-918fcaa50fdb&mode=design) to reflect this

Example of what this should look like:

**Note** This code is currently messy and may require some additional time to "clean up" or fix. 

Criteria:
- [ ] Make entire section a link

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #5790

Review URL: https://foundation-s-5790-large-qhqfpj.herokuapp.com/en/publication-page-with-child-article-pages/management-consider-left-plan-manager-thought/

I rejigged this code a bit to clean it up and make it a touch more manageable.

Criteria:
- [x] Make entire section a link

Preview:
arrows

### CODE REVIEW

**Comment by mmmavis** [comment]
@KalobTaulien @sabrinang 

I haven't done a full code review but I have some colours related questions

1. two custom greys are used as border colour. Any chance we can replace them by brand colours? 
  ```css
  --light-grey: rgb(204, 204, 204);
  --dark-grey: rgb(242, 242, 242);
  ```
2. Hover state of the arrow button has a11y issues. Can we give them a different design treatment?

"Previous/Next" label:
image

Link:
image

**Review by mmmavis — COMMENTED** [review]
@KalobTaulien @sabrinang 

I haven't done a full code review but I have some colours related questions

1. two custom greys are used as border colour. Any chance we can replace them by brand colours? 
  ```css
  --light-grey: rgb(204, 204, 204);
  --dark-grey: rgb(242, 242, 242);
  ```
2. Hover state of the arrow button has a11y issues. Can we give them a different design treatment?

"Previous/Next" label:
image

Link:
image

**Review by sabrinang — CHANGES_REQUESTED** [review]
- The hover state looks great!
- For 770-1200px viewports, can we reduce space in between so text is more centered?
image

- For mobile, the text is a bit out of alignment with other content now

image

**Comment by sabrinang** [comment]
@mmmavis thanks for the feedback and testing out a11y, I will look at the treatment closer

**Review by sabrinang — CHANGES_REQUESTED** [review]
From Mavis' feedback I looked at colors we can align and a11y, 
- for hover link we can use `$dark-blue` `#0d10bf` instead for better contrast that passess all WCAG contrast check
- for hover background, instead of a new gray we can use the same background colour as the summary box: `#e7e7fc` _(it still fails WCAG AAA but the summary section is also styled like this so I filed a [separate issue](https://github.com/mozilla/foundation.mozilla.org/issues/5950))_

**Comment by KalobTaulien** [comment]
@sabrinang New colors updated!

**Review by sabrinang — APPROVED** [review]
👍 Great, looks good

File: source/sass/components/article-page.scss [hunk 1]
```diff
@@ -444,33 +374,128 @@
...
   }
-  .article-nav-arrow--right::before {
-    right: 15px;
-    transform: initial;
+  &.article-link-right {
+    padding-left: var(--article-padding);
+    padding-right: initial;
   }
+}
+@media (max-width: 1200px) and (min-width: 770px) {
```
Comment (mmmavis) [inline source/sass/components/article-page.scss:427]: let's replace these numbers with variables

https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/variables.scss

File: source/sass/components/article-page.scss [hunk 2]
```diff
@@ -444,33 +374,128 @@
...
+  padding-top: 24px;
+  padding-bottom: 56px;
+
+  .body-large {
+    text-decoration: underline;
   }
 
-  .article-nav-arrow--left::before {
-    left: 15px;
+  &:hover {
```
Comment (mmmavis) [inline source/sass/components/article-page.scss:395]:
```suggestion
  @include hover-focus-active {
```

**Review by mmmavis — CHANGES_REQUESTED** [review]
two small requests!

**Review by mmmavis — APPROVED** [review]
Yay! Awesome work!

**Pull request merged**